### PR TITLE
DOCSP-35309 add Kafka to intro installation page

### PR DESCRIPTION
**Pull request by jocelyn-mendez1**
## DESCRIPTION
Add the Kafka deployment methods to the installation page table.

Tech Reviewer LGTM in comments below.

## STAGING
https://preview-mongodbjocelynmendez1.gatsbyjs.io/docs-relational-migrator/DOCSP-35309/installation/

## JIRA
https://jira.mongodb.org/browse/DOCSP-35309

## BUILD LOG
https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=65f4c2456fc5f04850fcba97

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/installation.txt [hunk 1]
```diff
@@ -20,28 +20,39 @@ For instructions on installing Relational Migrator, see the following pages:
...
+     - The Kafka with Confluent Cloud deployment method is an option for users who 
+       want the reliability of Kafka without having to manage their own cluster. 
+       Confluent is a fully-managed Kafka hosting provider run by the founders of 
+       Kafka. This deployment method uses APIs to add the newest Relational Migrator 
+       sink connectors to Confluent Cloud as managed connectors.
 
-     - The Kafka deployment model is not yet generally available, but you can 
-       reach out to your MongoDB account team to discuss private preview 
-       opportunities.
+       For more information, see :ref:`install-confluent-kafka`.
```
Comment (jocelyn-mendez1) [inline source/installation.txt:55]: [For reviewer]

This is causing a build error but will be resolved once https://github.com/mongodb/docs-relational-migrator/pull/96 is merged.

File: source/installation.txt [hunk 2]
```diff
@@ -20,28 +20,39 @@ For instructions on installing Relational Migrator, see the following pages:
...
 
-     - Suitable for larger production jobs. Integrates with your own Kafka 
-       infrastructure to provide enhanced resiliency.
+     - The Kafka deployment method is recommended for long-running Change Data 
+       Capture (CDC) syncs and large data migrations of more than 20GB. Apache Kafka 
+       is an open-source platform for distributed workloads that offers high 
+       availability and automatic recovery. This option is the most advanced, and is 
+       intended for users who require long-running or large production data syncs. 
+       
+       For more information, see :ref:`kafka-intro`.
```
Comment (jocelyn-mendez1) [inline source/installation.txt:45]: [For reviewer]

This is causing a build error but will be resolved once https://github.com/mongodb/docs-relational-migrator/pull/96 is merged.

File: source/installation.txt [hunk 3]
```diff
@@ -20,28 +20,39 @@ For instructions on installing Relational Migrator, see the following pages:
...
+       
+       For more information, see :ref:`Install on an Unattended Server 
+       <unattended-server>`. 
 
    * - Kafka
 
-     - Suitable for larger production jobs. Integrates with your own Kafka 
-       infrastructure to provide enhanced resiliency.
+     - The Kafka deployment method is recommended for long-running Change Data 
+       Capture (CDC) syncs and large data migrations of more than 20GB. Apache Kafka
```
Comment (ianf-mongodb) [inline source/installation.txt:40]:
```suggestion
       Capture (CDC) sync jobs and large data migrations of more than 20GB. Apache Kafka 
```

Comment (ianf-mongodb) [inline source/installation.txt:40]: **Suggestion**

Link for additional context

```suggestion
       Capture (CDC) syncs and large data migrations of more than 20GB. `Apache Kafka <https://kafka.apache.org/>`__
```

File: source/installation.txt [hunk 4]
```diff
@@ -20,28 +20,39 @@ For instructions on installing Relational Migrator, see the following pages:
...
+       availability and automatic recovery. This option is the most advanced, and is 
+       intended for users who require long-running or large production data syncs. 
+       
+       For more information, see :ref:`kafka-intro`.
+
+   * - Kafka with Confluent Cloud
+
+     - The Kafka with Confluent Cloud deployment method is an option for users who 
+       want the reliability of Kafka without having to manage their own cluster. 
+       Confluent is a fully-managed Kafka hosting provider run by the founders of
```
Comment (ianf-mongodb) [inline source/installation.txt:51]: **Suggestion**

Link for additional context:

```suggestion
       `Confluent <https://www.confluent.io/>`__ is a fully-managed Kafka hosting provider run by the founders of 
```

File: source/installation.txt [hunk 5]
```diff
@@ -20,28 +20,39 @@ For instructions on installing Relational Migrator, see the following pages:
...
+       
+       For more information, see :ref:`kafka-intro`.
+
+   * - Kafka with Confluent Cloud
+
+     - The Kafka with Confluent Cloud deployment method is an option for users who 
+       want the reliability of Kafka without having to manage their own cluster. 
+       Confluent is a fully-managed Kafka hosting provider run by the founders of 
+       Kafka. This deployment method uses APIs to add the newest Relational Migrator 
+       sink connectors to Confluent Cloud as managed connectors.
```
Comment (ianf-mongodb) [inline source/installation.txt:53]: **Question**

Should this be `sync`? Also will we have surface area in the docs for explaining the context of a sink/sync connector?

```suggestion
       sink connectors to Confluent Cloud as managed connectors.
```

**Review by ianf-mongodb — COMMENTED** [review]
Hi @jocelyn-mendez1 largely this looks good, just a couple small suggestions + 1 question for the edit. Thanks!

File: source/installation.txt (hunk 3 above)
Comment (jocelyn-mendez1) [reply source/installation.txt:40]: Updated.

Comment (jocelyn-mendez1) [reply source/installation.txt:40]: Updated.

File: source/installation.txt (hunk 4 above)
Comment (jocelyn-mendez1) [reply source/installation.txt:51]: Updated.

File: source/installation.txt (hunk 5 above)
Comment (jocelyn-mendez1) [reply source/installation.txt:53]: In this case it is `sink`. Currently we don't have anything major but I could add a sentence to the Kafka landing page where it defines sink connector as "an application that reads data from Kafka"?

**Comment by jocelyn-mendez1** [comment]
Thank you for the feedback @ianf-mongodb! Let me know of any other possible updates.

File: source/installation.txt (hunk 5 above)
Comment (ianf-mongodb) [reply source/installation.txt]: That SGTM, just a slight glossary type entry to denote what it is exactly should be fine.

File: source/installation.txt [hunk 6]
```diff
@@ -20,28 +20,41 @@ For instructions on installing Relational Migrator, see the following pages:
...
-
-     - The Kafka deployment model is not yet generally available, but you can 
-       reach out to your MongoDB account team to discuss private preview 
-       opportunities.
+     - The Kafka deployment method is recommended for long-running Change Data 
+       Capture (CDC) sync jobs and large data migrations of more than 20GB. `Apache 
+       Kafka <https://kafka.apache.org/>`__ is an open-source platform for distributed 
+       workloads that offers high availability and automatic recovery. This is the most 
+       advanced option that is intended for users who require long-running or large 
+       production data syncs.
```
Comment (ianf-mongodb) [inline source/installation.txt:44]: **Consistency**

```suggestion
       production data migrations. 
```

**Review by ianf-mongodb — APPROVED** [review]
Thanks for the updates @jocelyn-mendez1 this LGTM!

**Comment by tomhollander** [comment]
@jocelyn-mendez1 This looks good. One small ask: can we change the TOC entry "Installation on a single machine" to "Installation on a local machine" to match the text on this intro page? It's also more accurate since the unattended option is still a single machine.

**Comment by jocelyn-mendez1** [comment]
Thank you @tomhollander, I have updated all instances of "single machine" to "local machine" for consistency!

**Comment by davidsoulmdb** [comment]
Ask to mention time cutoff and downgrade recommendation to 'suitability' for now? Eg "The Kafka deployment method is **suitable** for long-running Change Data Capture (CDC) sync jobs and large data migrations of more than **4 weeks duration or 20GB database size**."

**Comment by davidsoulmdb** [comment]
+LGTM

**Pull request merged**